src/integrations/foursquare/index.js

```diff
@@ -0,0 +1,55 @@
+const axios = require('axios');
```

Comment:
Let's move this one to a class Foursquare API, similar to this one for example: https://gist.github.com/MaximAbramchuck/d99dc99b90fdc1d55e4aad060015c314